fix: DOCS-5395 (#962)
auth0/docs-v2: main/docs/authenticate/identity-providers/enterprise-identity-providers/active-directory-ldap/ad-ldap-connector/ad-ldap-connector-config-file-schema.mdx

| `LDAP_HEARTBEAT_SECONDS` | Time in seconds to keep the LDAP connection open. |   |
| `LDAP_SEARCH_ALL_QUERY` | The LDAP query used to list all users in the LDAP store. | `(objectCategory=person)` |
| `LDAP_SEARCH_GROUPS` | The LDAP query used to find groups in the LDAP store. For example: `(&(objectCategory=group)(member={0}))` | `(member:1.2.840.113556.1.4.1941:={0})` |

> | `LDAP_SEARCH_QUERY` | The LDAP query used to find users in the LDAP store. [Filters](https://github.com/auth0/ad-ldap-connector/blob/master/lib/users.js#L364) are required for the search to work as expected. If not configured, Auth0 does not send [blocked account](/docs/secure/attack-protection/brute-force-protection#email) notification emails. | `(&(objectCategory=person)(anr={0}))` |

Reviewer: ```suggestion
| `LDAP_SEARCH_QUERY` | The LDAP query used to find users in the LDAP store. This query requires [filters](https://github.com/auth0/ad-ldap-connector/blob/master/lib/users.js#L364) for the search to work correctly. If you do not configure filters, Auth0 does not send [blocked account](/docs/secure/attack-protection/brute-force-protection#email) notification emails. | `(&(objectCategory=person)(anr={0}))` |
```
active. didn't fix first sentence to align with others in the table.
